Face_Recognition/face_train.py

```diff
@@ -0,0 +1,57 @@
+# Face Recognition with OpenCV built in classifier
+# Aim: This script recognizes the face in an image using OpenCV's built-in recognizer. The vast application of this project is
+# recognizing the images in the running videos.
+# Feature: Easy and understandable code
+
+import os
+import cv2 as cv
+import numpy as np
+
+people = []
+DIR = r'images'  # DIR - path of the folder containing images
+for i in os.listdir(DIR):  # Will append the names of the person in the people list
+    people.append(i)
+
+haar_cascade = cv.CascadeClassifier('haar_face.xml')
+
+# For every face in the list the corresponding labels to the features are designated.
+features = []
+labels = []
+
+def train_face(features):
```

Comment:
```suggestion
def train_face(features:list):
```

Reply:
@aakriti1318 As I mentioned earlier , the issue is that the features argument is being treated as a ndarray and you are trying to call append function which is causing the error. So just explicitly define its type as list

Reply:
Yes, that's the issue, the features and labels are called twice once as a list and then as ndarray. That's why Pylint is trying to refer it as ndarray but when it gets append member it causes error

Reply:
@kaustubhgupta Exactly

Reply:
I have changed my entire code still not working.